tequilapi/endpoints/payout.go

```diff
@@ -123,6 +130,57 @@ func (endpoint *payoutEndpoint) UpdatePayoutInfo(resp http.ResponseWriter, reque
 	err = endpoint.payoutInfoRegistry.UpdatePayoutInfo(
 		id,
 		payoutInfoReq.EthAddress,
+		endpoint.signerFactory(id),
+	)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusInternalServerError)
+		return
+	}
+
+	resp.WriteHeader(http.StatusOK)
+}
+
+// swagger:operation PUT /identities/{id}/referral Identity updateReferralInfo
+// ---
+// summary: Registers referral info
+// description: Registers referral code for identity
+// parameters:
+// - name: id
+//   in: path
+//   description: Identity stored in keystore
+//   type: string
+//   required: true
+// - in: body
+//   name: body
+//   description: Parameter in body (referral_code) is required
+//   schema:
+//     $ref: "#/definitions/ReferralInfoDTO"
+// responses:
+//   200:
+//     description: Referral info registered
+//   400:
+//     description: Bad request
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+//   422:
```

Comment:
We are sending only `http.StatusBadRequest` in the function, no 422 and 500.

Reply:
it can return 500 in line https://github.com/mysteriumnetwork/node/pull/1165/files#diff-76a2d4645eb783ed4db8078ac22056fbR184

Reply:
Removed the 422 though :)